### ISSUE TITLE
Parsing fails with regex 0.1.59

### ISSUE THREAD

**Issue by afck** [docopt/docopt.rs#172]
The latest regex update broke usage string parsing in some of our crates (crust, routing), and three of docopt's tests:
```
---- test::testcases::test_120_testcases stdout ----
	thread 'test::testcases::test_120_testcases' panicked at 'Invalid usage: Argument 'variant.' is not of the form ARG or <arg>.', src/test/mod.rs:7

---- test::testcases::test_121_testcases stdout ----
	thread 'test::testcases::test_121_testcases' panicked at 'Invalid usage: Argument 'variant.' is not of the form ARG or <arg>.', src/test/mod.rs:7

---- test::testcases::test_122_testcases stdout ----
	thread 'test::testcases::test_122_testcases' panicked at 'Invalid usage: Argument 'variant.' is not of the form ARG or <arg>.', src/test/mod.rs:7
```

**Comment by BurntSushi** [comment]
Thanks @afck! This is definitely a bug in the latest regex update. I'll have a fix out today.

**Comment by BurntSushi** [comment]
@afck This should be fixed in `regex 0.1.60` on crates.io. Just run a `cargo update`.
